src/Raven.Server/Documents/Handlers/Processors/SmugglerValidationOptionsProcessor.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Raven.Server.Documents.Patch;
+using Raven.Server.Json;
+using Raven.Server.Web;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Handlers.Processors
+{
+    internal class SmugglerValidationOptionsProcessor<T> : AbstractHandlerProcessor<RequestHandler, T>
```

Comment:
SmugglerHandlerProcessorForValidateOptions 
Also rename method to ValidateOptions